projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -478,123 +500,134 @@ describe('DatagridComponent', () => {
                             pageSizeOptions: undefined,
                         };
                         this.finder.detectChanges();
-                        expect(this.clrGridWidget.getPaginationSizeSelectorText()).toEqual('Total Items5');
+                        expect(this.clrGridWidget.getPaginationSizeSelector().text()).toEqual('Total Items5');
                     });
                 });
 
                 describe('shouldShowPageSizeSelector', () => {
                     it('hides the dropdown when set to false', function (this: HasFinderAndGrid): void {
-                        this.finder.hostComponent.pagination = {
-                            ...this.finder.hostComponent.pagination,
+                        this.hostComponent.pagination = {
+                            ...this.hostComponent.pagination,
                             shouldShowPageSizeSelector: false,
                         };
                         this.finder.detectChanges();
-                        expect(this.clrGridWidget.getPaginationSizeSelectorText()).toEqual('');
+                        expect(this.clrGridWidget.getPaginationSizeSelector().length()).toEqual(0);
                     });
 
                     it('shows the dropdown when set to true', function (this: HasFinderAndGrid): void {
-                        this.finder.hostComponent.pagination = {
-                            ...this.finder.hostComponent.pagination,
+                        this.hostComponent.pagination = {
+                            ...this.hostComponent.pagination,
                             shouldShowPageSizeSelector: true,
                         };
                         this.finder.detectChanges();
-                        expect(this.clrGridWidget.getPaginationSizeSelectorText()).toEqual('Total Items52050100');
+                        expect(this.clrGridWidget.getPaginationSizeSelector().text()).toEqual('Total Items52050100');
                     });
                 });
             });
 
             describe('@Input() paginationDropdownText', () => {
                 it('displays the pagination dropdown information on page one', function (this: HasFinderAndGrid): void {
-                    this.finder.hostComponent.pagination = {
-                        ...this.finder.hostComponent.pagination,
+                    this.hostComponent.pagination = {
+                        ...this.hostComponent.pagination,
                         shouldShowPageSizeSelector: true,
                     };
                     this.finder.detectChanges();
-                    expect(this.clrGridWidget.getPaginationSizeSelectorText()).toEqual('Total Items52050100');
+                    expect(this.clrGridWidget.getPaginationSizeSelector().text()).toEqual('Total Items52050100');
                 });
             });
 
             describe('@Input() height', () => {
+                function getGridHeight(element: TestElement): string {
```

Comment:
Good use of a helper function! Should we add some function to the test Element like `getStylePropertyValue(key: string)` to implement this instead?